Add AOT /1 (#34320)
dotnet/AspNetCore.Docs: aspnetcore/fundamentals/openapi/aspnetcore-openapi.md

### Customizing run-time behavior during build-time document generation

> Build-time OpenAPI document generation functions by launching the apps entrypoint with a mock server implementation. A mock server is required to produce accurate OpenAPI documents because all information in the OpenAPI document can't be statically analyzed. Because the apps entrypoint is invoked, any logic in the apps startup is invoked. This includes code that injects services into the [DI container](xref:fundamentals/dependency-injection) or reads from configuration. In some scenarios, it's necessary to restrict the code paths that will run when the apps entry point is being invoked from build-time document generation. These scenarios include:

Reviewer: ```suggestion
Build-time OpenAPI document generation functions by launching the app's entry point with a mock server implementation. A mock server is required to produce accurate OpenAPI documents because some of the information in the OpenAPI document can't be statically analyzed. Because the app's entry point is invoked, any logic in the app's startup is invoked. This includes code that injects services into the [DI container](xref:fundamentals/dependency-injection) or reads from configuration. In some scenarios, it's necessary to restrict the code paths that will run when the apps entry point is being invoked from build-time document generation. These scenarios include:
```
